docs/book/driver-deployment/installation.md

```diff
@@ -61,7 +61,7 @@ user = "<username>"
 password = "<password>"
 port = "<port>"
 ca-file = <ca file path> # optional, use with insecure-flag set to false
-datacenters = "<datacenter-name1>, <datacenter-name2>, ..."
+datacenters = "<datacenter-path>, <datacenter-path>, ..."
```

Comment:
Can you change it to datacenter1-path, datacenter2-path and so on

Reply:
fixed.